VMBackup/main/guestsnapshotter.py

```diff
@@ -252,6 +264,8 @@ def snapshotall_parallel(self, paras, freezer, thaw_done, g_fsfreeze_on):
         except Exception as e:
             errorMsg = " Unable to perform parallel snapshot with error: %s, stack trace: %s" % (str(e), traceback.format_exc())
             self.logger.log(errorMsg)
+            self.logger.log("Setting to sequential snapshot")
+            self.hutil.set_value_to_configfile('seqsnapshot', '1')
```

Comment:
exception can be due to many reasons, we should not set to Seq here. We should have try-catch around the exact code where we know it is due to mp-process.